pkg/storages/azure/folder.go

```diff
@@ -124,7 +133,7 @@ func ConfigureFolder(prefix string, settings map[string]string) (storage.Folder,
 
 	var containerUrlString string
 	var containerClient azblob.ContainerClient
-	if usingToken {
+	if authType == AzureSASTokenAuth {
```

Comment:
Now it looks like a good place to extract the logic inside the `if` branches into separate functions to improve readability for future contributors.